L/liblsl/build_tarballs.jl

```diff
@@ -0,0 +1,77 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "liblsl"
+version = v"1.13.0"
+
+# Collection of sources required to complete build
+sources = [
+    "https://github.com/sccn/liblsl/archive/1.13.0.tar.gz" =>
+    "5b304a5365eba33852da96badfbd9d66556caf4a00c87947a59df2942680a617",
+    DirectorySource("./patches"),
```

Comment:
Other important warning from audit:
```
┌ Warning: Unable to find valid license file in "${prefix}/share/licenses/liblsl"
└ @ BinaryBuilder /depot/packages/BinaryBuilder/xbKKQ/src/Auditor.jl:416
```
Note that the license will automatically be installed if you move ` L/liblsl/patches` to ` L/liblsl/bundled/patches` and here you use
```suggestion
    DirectorySource("./bundled"),
```
This is mostly for historical reasons.